app/components/Layout/Footer.tsx

```diff
@@ -7,10 +8,16 @@ const Footer = () => {
         <div className="container">
           <ul>
             <li>
-              <a href=".">Home</a>
+              <Link href="/resources/disclaimer">Disclaimer</Link>
             </li>
             <li>
-              <a href=".">Contact Us</a>
```

Comment:
I don't think those should be removed. `Home` should be a link to `/`, and you can add another page for `Contact Us`

Reply:
Added links and Contact Us page.